Surveys/Sources/Supports/Helpers/Typealiases/Typealiases.swift

```diff
@@ -13,3 +13,4 @@ typealias CancelBag = Set<AnyCancellable>
 typealias HTTPHeaders = [String: String]
 typealias Localize = R.string.localizable
 typealias Observable<T> = AnyPublisher<T, Error>
+typealias Driver<T> = AnyPublisher<T, Never>
```

Comment:
I suggest adding comment for Observable and Driver for quick documentation.
```suggestion
/// Publisher with error `AnyPublisher<T, Error>`
typealias Observable<T> = AnyPublisher<T, Error>
/// Publisher with no error `AnyPublisher<T, Never>`
typealias Driver<T> = AnyPublisher<T, Never>
```

Reply:
Good suggestion.